api/src/controllers/cart.js

```diff
@@ -0,0 +1,148 @@
+/* eslint linebreak-style: ["error", "windows"] */
+const {
+  Cart,
+  User,
+  Orderline,
+  Product,
+} = require('../models/index');
+
+const addItem = async (req, res) => {
+  // primero busco el usuario en la base de datos
+  const user = await User.findOne({
+    where: {
+      id: req.body.id,
+    },
+    include: Cart,
+  });
+  // si no lo encuentro mando un error
+  if (user == null) {
```

Comment:
Se usa doble igual en lugar de triple igual. Recordemos activar el eslint para así tener un código uniforme.

Reply:
Expected '===' and instead saw '=='.eslinteqeqeq , tengo el eslint activado y me pedia eso